drivers/SmartThings/philips-hue/src/handlers/lifecycle_handlers/init.lua

```diff
@@ -135,12 +135,15 @@ local valid_migration_kwargs = {
   "force_migrate_type"
 }
 function LifecycleHandlers.migrate_device(driver, device, ...)
-  local migration_kwargs = select(-1, ...)
   local opts = {}
-  if type(migration_kwargs) == "table" then
-    for _, key in ipairs(valid_migration_kwargs) do
-      if migration_kwargs[key] then
-        opts[key] = migration_kwargs[key]
+  local args = {...}
```

Comment:
While it might not cause any actual issues with this specific situation, it is often better to use `table.pack(...)` because it accounts for `nil` elements that occur before the end of the variadic elements.

`{...}` will stop at the first `nil` element it encounters.